functions/go/policy-controller-validate/validate.go

```diff
@@ -151,7 +131,7 @@ func parseResults(results []*opatypes.Result) error {
 			return fmt.Errorf("could not cast to unstructured: %+v", r.Resource)
 		}
 		name := u.GetName()
-		path, found := u.GetAnnotations()[constants.SourcePathAnnotation]
+		path, found := u.GetAnnotations()[SourcePathAnnotation]
```

Comment:
our library provides utilities for this. Each functions shouldn't need to define this annotation constant and perform this low-level operation.

Reply:
I updated it to use the SourcePathAnnotation definition from kyaml.
[`Result.Resource` is a `unstructured.Unstructured`](https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/197/files/245c2066d786a31434de2978df02bf2906eafdd0..c636365a5b120b6b2d5cbb6e3d32b7283700a403#diff-a80d7be973fa160be84208745ee7f48d395becec2919f109f89d8fe3d14a8fe5R130). If we want to use the library from kyaml, we need to convert `unstructured.Unstructured` to `RNode` which requires one more round of marshal and unmarshal. It will take more time and I don't think it will make the code more readable.